win_installer/cppcheck.wxs

```diff
@@ -54,18 +54,6 @@
               <File Id='qwindowsdll' Name='qwindows.dll' Source='$(var.QtDllDir)\platforms\qwindows.dll' />
             </Component>
           </Directory>
-          <Directory Id='QtStylesFolder' Name='styles'>
-            <Component Id='QtStyles' DiskId='1' Guid='$(var.qtstylesGUID)'>
-              <File Id='qwindowsvistastyledll' Name='qwindowsvistastyle.dll' Source='$(var.QtDllDir)\styles\qwindowsvistastyle.dll' />
```

Comment:
hmm.. I have the feeling this used to be needed otherwise the GUI would not look very nice. I assume you have executed the gui without the dll and checked that dialogs etc look OK.

Reply:
I did not see any differences. I also did not see the DLL being loaded. Will check again.

Reply:
I checked again and the file is actually being loaded. Will restore.